interface/reports/appointments_report.php

```diff
@@ -322,7 +322,7 @@ function refreshme() {
                                 <a href='#' class='btn btn-secondary btn-print' id='printbutton'>
                                     <?php echo xlt('Print'); ?>
                                 </a>
-                                <button class="btn btn-default btn-transmit"><?php echo xlt('Export CSV'); ?></button>
+                                <button name="appointment_report" class="btn btn-default btn-transmit" id="csv-report"><?php echo xlt('Export CSV'); ?></button>
                                 <a href='#' class='btn btn-secondary btn-transmit' onclick='window.open("../patient_file/printed_fee_sheet.php?fill=2", "_blank").opener = null' onsubmit='return top.restoreSession()'>
```

Comment:
using the name element, the system is able to pass the name of the report so when the file is generated, it can reflect accurately the name of the report.  This way it does not have to be a generic name report.csv.